### PR TITLE
Draft of Get Started page

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
@michel-kraemer How do i add an image? I want to try to put a screenshot of the vert.x starter

**Comment by michel-kraemer** [comment]
@slinkydeveloper Create a directory `public/images` and copy the image there. Add the image in your Markdown file as follows:

    Fancy starter image

(Every file in the `public` directory will be copied as is).

**Comment by slinkydeveloper** [comment]
Screenshot_2020-04-03 Vert x Get started

File: pages/get-started.mdx [hunk 1]
```diff
@@ -7,6 +7,98 @@ export const meta = {
...
 
-TODO
+In this guide you'll learn how to get started with a new Vert.x project.
 
-test (Julien)
+Before starting, you need:
+
+* A JDK 1.8 or higher
+* A text editor or IDE
+* Maven
```
Comment (rdruss) [inline pages/get-started.mdx]: Do you need to have a version for Maven?

Comment (slinkydeveloper) [reply pages/get-started.mdx]: I suppose Maven 3+ should be fine, let me add it

**Comment by slinkydeveloper** [comment]
Now the example uses Vert.x Web and returns a Json (IMO It's nice to show how easy is to create a JsonObject in Vert.x)

Screenshot_2020-04-10 Vert x Get started

**Comment by slinkydeveloper** [comment]
@michel-kraemer can we have a tabbed view of the output of the terminal? To show or the output of `http` or the output of `curl`

**Pull request merged**